gradle/versions.gradle

```diff
@@ -11,7 +11,7 @@ ext {
 
 // Plugins
 ext.plugs = [
-    "gradle"              : "com.android.tools.build:gradle:3.0.0-beta6",
+    "gradle"              : "com.android.tools.build:gradle:3.0.0",
```

Comment:
Can you undo this change because we are going most likely use 3.1 from this PR: https://github.com/KeepSafe/dexcount-gradle-plugin/pull/225.

Reply:
done